model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/RealmCacheSession.java

```diff
@@ -546,6 +546,42 @@ static String getRoleByNameCacheKey(String container, String name) {
         return getClientDelegate().getClients(realm);
     }
 
+    @Override
+    public List<ClientModel> getAlwaysDisplayInConsoleClients(RealmModel realm) {
```

Comment:
I don't yet have enough expertise in this area (caching etc.) to properly review this, so I've got a few stupid questions. :)

Why not to directly call `getClientDelegate().getClients(realm)` like here?
https://github.com/keycloak/keycloak/blob/45ea5bd34828cefcd7301d168e40b854ff52234f/model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/RealmCacheSession.java#L544-L547

Reply:
I don't have expertise either; if getClientDelegate would be better I can change it.